ecl/resource_ecl_network_load_balancer_v2.go

```diff
@@ -0,0 +1,1384 @@
+		Delay:        5 * time.Second,
+		PollInterval: loadBalancerCreatePollInterval,
+		MinTimeout:   10 * time.Second,
+	}
+
+	_, err = stateConf.WaitForState()
+	if err != nil {
+		return fmt.Errorf(
+			"error waiting for load balancer (%s) to become ACTIVE: %s",
+			loadBalancer.ID, err)
+	}
+
+	// also wait for Load Balancer Interfaces
+	for _, lbIF := range loadBalancer.Interfaces {
+		stateConf = &resource.StateChangeConf{
+			Pending:      []string{"MONITORING_UNAVAILABLE", "PENDING_UPDATE"},
+			Target:       []string{"ACTIVE", "DOWN"},
+			Refresh:      waitForLoadBalancerInterfaceComplete(networkClient, lbIF.ID),
+			Timeout:      d.Timeout(schema.TimeoutUpdate),
+			Delay:        5 * time.Second,
+			PollInterval: loadBalancerUpdatePollInterval,
+			MinTimeout:   3 * time.Second,
+		}
+
+		_, err = stateConf.WaitForState()
+		if err != nil {
+			return fmt.Errorf(
+				"error waiting for Load Balancer Interface (%s) to become ACTIVE(after Load Balancer create): %s",
+				lbIF.ID, err)
+		}
+	}
+
+	// If interface configs are specified, update interfaces according to the configs.
+	interfaceConfigs := d.Get("interfaces").([]interface{})
+	if len(interfaceConfigs) > 0 {
+
+		for configIndex, v := range interfaceConfigs {
+			interfaceConfig := v.(map[string]interface{})
+
+			updateInterfaceOpts := getLoadBalancerInterfaceInitialUpdateOpts(interfaceConfig)
+
+			// .. update, call Show interface API and wait for active
+			err = updateLoadBalancerInterface(networkClient, d, loadBalancer.Interfaces[configIndex].ID, *updateInterfaceOpts)
+			if err != nil {
+				return err
+			}
+		}
+
+		// Update default_gateway if specified
+		if v, ok := d.GetOk("default_gateway"); ok {
+			s := v.(string)
+			var defaultGateway interface{}
+			if s == "" {
+				defaultGateway = nil
+			} else {
+				defaultGateway = s
+			}
+			updateOpts := load_balancers.UpdateOpts{
+				DefaultGateway: &defaultGateway,
+			}
+			// update, call Show load_balancer API and wait for active
+			err = updateLoadBalancer(networkClient, d, loadBalancer.ID, updateOpts)
+			if err != nil {
+				return err
+			}
+		}
+	}
+
+	// If syslog server configs are specified, create syslog servers according to the configs.
+	syslogConfigs := d.Get("syslog_servers").([]interface{})
+	for _, v := range syslogConfigs {
+		syslogConfig := v.(map[string]interface{})
+
+		err := createLoadBalancerSyslogServer(syslogConfig, loadBalancer.ID, networkClient, d)
+		if err != nil {
+			return err
+		}
+
+	}
+
+	return resourceNetworkLoadBalancerV2Read(d, meta)
+}
+
+func resourceNetworkLoadBalancerV2Read(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	networkClient, err := config.networkV2Client(GetRegion(d, config))
+	if err != nil {
+		return fmt.Errorf("error creating ECL network client: %s", err)
+	}
+
+	loadBalancer, err := load_balancers.Get(networkClient, d.Id()).Extract()
+	if err != nil {
+		return CheckDeleted(d, err, "load-balancer")
```

Comment:
エラーメッセージをもう少し詳細にしていただけますか？
```suggestion
		return CheckDeleted(d, err, "error getting ECL load balancer")
```

Reply:
Suggestionの通り、詳細化しました。他1箇所CheckDeletedを使用している箇所があったため、同様に修正しています。